src/common/utils.cpp

```diff
@@ -818,6 +818,43 @@ std::string trim(std::string const& str, std::string const& whitespace)
     return str.substr(strBegin, strRange);
 }
 
+/// Matches the given target with the given pattern, using the algorithm linked below.
+/// There's possibly a more clever / modern c way of doing this.
```

Comment:
No need for this comment, if you wanted to leave behind some thoughts, you could mention that this could also be achieved in regex

Reply:
Removed